### PR TITLE
LPS-137517

### PR DESCRIPTION
**Pull request by rodrigopaulino**
Related ticket: https://issues.liferay.com/browse/LPS-137517

### CODE REVIEW

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by rodrigopaulino** [comment]
ci:reevaluate:1297401_348

**Comment by rodrigopaulino** [comment]
ci:test:relevant

**Comment by natocesarrego** [comment]
Your pull seems good to me. :slightly_smiling_face: Once it touches on front-end code too then I've already requested Milton to double check.

Thanks.

File: modules/apps/data-engine/data-engine-taglib/test/mock/fields.es.js [hunk 1]
```diff
@@ -157,6 +157,10 @@ export const FIELDS = [
 		type: 'document_library',
 		value: 'document_library',
 	},
+	{
+		rulesActionDisabled: true,
+		rulesConditionDisabled: true,
+	},
```
Comment (natocesarrego) [inline modules/apps/data-engine/data-engine-taglib/test/mock/fields.es.js:163]: @rodrigopaulino, can you please add a unit test [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/data-engine/data-engine-taglib/test/js/data-layout-builder/components/rules/editor/Editor.es.js) instead?

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**